starter-code/src/components/Contacts.js

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import contacts from "../contacts.json";
+
+export class Contacts extends Component {
+  state = {
+    firstFive: contacts.slice(0, 5)
+  };
+
+  addNew = () => {
+    const random = Math.floor(Math.random() * 200);
```

Comment:
multiplying by 200 works this time, but the length of the array could change so it's better to times it by the length of the `contacts`